addons/compat_ace_fire/config.cpp

```diff
@@ -0,0 +1,49 @@
+#include "script_component.hpp"
+
+#pragma hemtt flag pe23_ignore_has_include
+#if __has_include("\z\ace\addons\fire\script_component.hpp")
+#define PATCH_SKIP "ACE Fire"
+#endif
+
+#ifndef PATCH_SKIP
```

Comment:
why not just use the else case here, ace fire not loaded will just skip this config completely, including the pre init and pre start eventhandlers

Reply:
more clarification, when this is added in the cfgpatches child `skipWhenMissingDependencies = 1;`

Reply:
It's how ACE has their "No Medical" medical skip set up, so I was just copying that.